scripts/dev.js

```diff
@@ -13,6 +13,8 @@ const PROJECT_ROOT = getProjectRoot()
 const TOOLBOX_ROOT = path.join(PROJECT_ROOT, 'scripts', 'toolbox')
 const pgMigrate = require('node-pg-migrate').default
 const cliPgmConfig = require('../packages/server/postgres/pgmConfig')
+const {generate} = require('@graphql-codegen/cli')
+const yaml = require('js-yaml')
```

Comment:
we don't have js-yaml installed in the app, we'll either need to install it (if you want to keep the yaml) or we should build the config as a JS object (might be cleaner?). the reason this works today is because we have a dependency that already includes it, but if that dependency is removed, then this call will mysteriously break.

we'll also need a way to call this from CI. perhaps add the same call to prod.js?